### PR TITLE
Reduce external calls made in head

### PR DESCRIPTION
**Pull request by jdamiba**
closes https://github.com/plotly/documentation/issues/1527

The purpose of this PR is to reduce the number of run-time calls to external stylesheets that are made. 

[x] - combine the two google font calls into one 
[x] - remove calls to `//plot.ly/gh-pages/documentation/static` with references to `{{site.staticurl}}`
[x] - remove unneeded call to cloudflare for code highlighting javascript
[x] - remove duplicate imports in `main.css`

A by-product of these changes is some slight variations in how code is highlighted.

### CODE REVIEW

File: all_static/css/main.css [hunk 1]
```diff
@@ -1,6 +1,4 @@
 @import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,600,600italic,700,700italic,800,800italic);
```
Comment (nicolaskruchten) [inline all_static/css/main.css]: can we get rid of this top import also plz?

File: all_static/css/main.css [hunk 2]
```diff
@@ -1,6 +1,4 @@
...
-@import url(https://help.plot.ly/stylesheets/plotly_icons.css);
```
Comment (nicolaskruchten) [inline all_static/css/main.css]: these were just unused or... ?

File: _includes/layouts/head.html [hunk 3]
```diff
@@ -4,31 +4,23 @@
...
       <!-- code highlighting -->
-      <link rel="stylesheet" href="//plot.ly/gh-pages/documentation/static/javascripts/codehighlight/styles/tomorrow.css">
-      <link rel="stylesheet" type="text/css" href="//plot.ly/gh-pages/documentation/static/css/js-splash-highlight.css"/>
+      <link rel="stylesheet" href="{{site.staticurl}}/javascripts/codehighlight/styles/tomorrow.css">
+      <link rel="stylesheet" type="text/css" href="{{site.staticurl}}/css/js-splash-highlight.css"/>
 
       <!-- Main Stylesheets -->
-      <link rel="stylesheet" type="text/css" href="https://images.plot.ly/assets/css/normalize.css">
-      <link rel="stylesheet" type="text/css" href="{{site.staticurl}}/css/main.css?version={{ site.data.cache_bust_css | truncate: 10, '' }}">
+      <link rel="stylesheet" type="text/css" href="{{site.staticurl}}/css/normalize.css">
```
Comment (nicolaskruchten) [inline _includes/layouts/head.html]: we should be able to concatenate all of these CSS files into one!

File: all_static/css/main.css (hunk 1 above)
Comment (jdamiba) [reply all_static/css/main.css]: This being built into the `main.css` file as a result of being included in the `abstract.scss` file. Removing the import from there removes it from here.

File: all_static/css/main.css (hunk 2 above)
Comment (jdamiba) [reply all_static/css/main.css]: Yes, I couldn't find any references to these icons in this repo.

File: .circleci/config.yml [hunk 4]
```diff
@@ -54,7 +54,7 @@ jobs:
             mkdir snapshots
             cd _site
             cp -r 'all_static' '../snapshots'
-            cp 'index.html' '../snapshots'
```
Comment (nicolaskruchten) [inline .circleci/config.yml]: aha i'll bet we were previously snapshotting the file from prod ;)

Comment (jdamiba) [reply .circleci/config.yml]: Yes, this is why the screenshot of the main index page was blank.

**Comment by jdamiba** [comment]
superseded by https://github.com/plotly/documentation/pull/1547

**Pull request closed without merging**